### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Ram81**
### Description

This PR - 

- [x] Updates all dependencies to the latest stable version after dajngo 2 migration.
- [x] Not updating moto because of this [issue](https://github.com/getsentry/responses/issues/358)

### CODE REVIEW

**Comment by Ram81** [comment]
Not updating moto because of this [issue](https://github.com/getsentry/responses/issues/358)

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
@Ram81 Could you please resolve the conflicts here?

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
@Ram81 Could you please resolve the conflicts here?

**Comment by Ram81** [comment]
Sure, will do

File: settings/common.py [hunk 1]
```diff
@@ -61,6 +61,7 @@
 THIRD_PARTY_APPS = [
     "allauth",
     "allauth.account",
+    'allauth.socialaccount',
```
Comment (RishabhJain2018) [inline settings/common.py:64]: Why are we adding this app?

File: requirements/worker.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
-networkx==2.1
-pandas==0.25.3
-pycocotools==2.0.0
-scipy==1.5.2
+matplotlib==3.3.3
+networkx==2.5
+pandas==1.1.4
+pycocotools==2.0.2
+scipy==1.5.4
 sklearn==0.0
```
Comment (RishabhJain2018) [inline requirements/worker.txt:6]: Can we please change it to some relevant version than `0.0`?

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

File: settings/common.py (hunk 1 above)
Comment (Ram81) [reply settings/common.py:64]: After upgrading the allauth package we had to add this app explicitly to get it working. I can't find the git issue where I found the solution. I will update if I find it here

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**